lib/commands/sdk.js

```diff
@@ -175,83 +159,64 @@ async function list(logger, config, cli) {
 	}
 
 	const defaultInstallLocation = cli.env.installPath;
-	const locations = cli.env.os.sdkPaths.map(p => afs.resolvePath(p));
-	if (!locations.includes(defaultInstallLocation)) {
-		locations.push(defaultInstallLocation);
-	}
-
-	let customLocations = config.get('paths.sdks');
-	if (customLocations) {
-		if (!Array.isArray(customLocations)) {
-			customLocations = [ customLocations ];
-		}
-		customLocations.forEach(location => {
-			location = afs.resolvePath(location);
-			if (location && !locations.includes(location) && fs.existsSync(location)) {
-				locations.push(location);
-			}
-		});
-	}
+	const locations = Array.from(
+		new Set([
+			cli.env.os.sdkPaths,
+			defaultInstallLocation,
+			config.get('paths.sdks')
+		].flat().map(p => p && afs.resolvePath(p)).filter(Boolean))
+	).sort();
 
 	if (cli.argv.output === 'json') {
-		Object.keys(sdks).forEach(function (ver) {
+		for (const ver of vers) {
 			delete sdks[ver].commands;
 			delete sdks[ver].packageJson;
 			delete sdks[ver].platforms;
-		});
-
-		// turn Map<string, Release> into bare object we used to report
-		const releasesObj = {};
-		if (releases) {
-			releases.forEach((value, key) => releasesObj[key] = value.url);
 		}
-		var obj = {
+
+		const obj = {
 			activeSDK,
+			branch: branchBuilds.length ? {
```

Comment:
`branchBuilds` can be undefined here so need to guard against that
```suggestion
			branch: branchBuilds?.length ? {
```

Reply:
Fixed